CODE_OF_CONDUCT.md

```diff
@@ -6,7 +6,7 @@ In the interest of fostering an open and welcoming environment, we as
 contributors and maintainers pledge to making participation in our project and
 our community a harassment-free experience for everyone, regardless of age, body
 size, disability, ethnicity, sex characteristics, gender identity and expression,
-level of experience, education, socio-economic status, nationality, personal
+level of experience, education, socioeconomic status, nationality, personal
```

Comment:
Assuming this is the American English?

British English seems to be `socio-economic`

Reply:
I installed the American dictionary and this does seem to be the case, feel free to resolve this one.